core/driver/sources/src/main/scala/org/virtuslab/ideprobe/IntelliJFixture.scala

```diff
@@ -129,17 +129,20 @@ final case class IntelliJFixture(
 
 object IntelliJFixture {
   private val ConfigRoot = "probe"
+  lazy val defaultIdeProbeConfig: IdeProbeConfig = readIdeProbeConfig(Config.fromReferenceConf, "probe")
```

Comment:
can we just call it `default config`? Also use `ConfigRoot` rather than explicitly "probe" string.

Reply:
fixed as suggested